packages/stockflux-launcher/src/titlebar/Titlebar.js

```diff
@@ -0,0 +1,67 @@
+import cx from 'classnames';
+import { ScreenEdge } from 'openfin-react-hooks';
+import React, { useCallback } from 'react';
+import { OpenfinApiHelpers } from 'stockflux-core';
+import Components from 'stockflux-components';
+import './Titlebar.css';
+
+export default ({ dockedTo }) => {
+  const onMinimizeClick = useCallback(async () => {
+    const currentWindow = await OpenfinApiHelpers.getCurrentWindow();
+    currentWindow.minimize();
+  }, []);
+
+  const onCloseClick = useCallback(async () => {
+    const currentWindow = await OpenfinApiHelpers.getCurrentWindow();
+    currentWindow.close();
+  }, []);
+
+  const dockedHorizontal = [ScreenEdge.NONE, ScreenEdge.TOP].includes(dockedTo);
+
+  const ButtonComponent = (closeButton, onClick, title) => {
+    return (
+      <div
+        className={cx(
+          'column column-right header-icon',
+          dockedHorizontal ? 'add-padding-right' : ''
+        )}
+        onClick={onClick}
+        title={title}
+      >
+        {closeButton && <Components.Icons.Toolbar.CloseIcon />}
+        {!closeButton && <Components.Icons.Toolbar.MinimizeIcon />}
```

Comment:
Not sure this is loading the correct svg, as both close and minimise icons are the same. 
![image](https://user-images.githubusercontent.com/49903895/68019650-4c7b0200-fc94-11e9-98df-c7227160c79b.png)

Reply:
Good catch; this is a typo on the import in the `components` folder that I missed